plugins/faustwp/.docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG WP_VERSION=latest
+ARG WP_VERSION=6.4
```

Comment:
Not a stopper, but let's add a to-do for later on this... just in case.

Reply:
https://github.com/wpengine/faustjs/issues/1875